### PR TITLE
Bug 1148068 - Indicate in tooltip that a run has been retriggered

### PR DESCRIPTION
**Pull request by MikeLing**
screenshot from 2015-07-08 10 03 50

[Review on Reviewable](https://reviewable.io/reviews/mozilla/treeherder/717)

### CODE REVIEW

File: ui/js/graphs.js [hunk 1]
```diff
@@ -84,6 +84,12 @@ perf.controller('GraphsCtrl', [
                                      function(resultSetId) {
                                        return (resultSetId < dataPoint.resultSetId);
                                      });
+        var triggerCounter = _.countBy(phSeries.flotSeries.resultSetData,
```
Comment (tojonmz) [inline ui/js/graphs.js]: Wondering if we might want to use the term 'retrigger' generally where applicable in this PR? If nothing else for UI association purposes with Treeherder. eg. `var retriggerNum =`, or something :)

File: ui/js/graphs.js [hunk 2]
```diff
@@ -84,6 +84,12 @@ perf.controller('GraphsCtrl', [
...
+                                      function(resultSetId) {
+                                          return resultSetId === dataPoint.resultSetId ? 'trigger':'untrigger';
```
Comment (tojonmz) [inline ui/js/graphs.js]: Also here, wondering perhaps about `'retrigger':'original',` (if untrigger is the first instance of the job).

File: ui/partials/perf/graphsctrl.html [hunk 3]
```diff
@@ -78,6 +78,7 @@
         </p>
         <p ng-hide="tooltipContent.revision">Loading revision...</p>
         <p id="tt-t" class="small" ng-bind="tooltipContent.date"></p>
+        <p id="tt-v" ng-show="tooltipContent.retriggered > 0">Retriggered: {{tooltipContent.retriggered}}</p>
```
Comment (tojonmz) [inline ui/partials/perf/graphsctrl.html]: Similarly here, wondering about `Retriggers:` and `.retriggers` for these, at least as far as how they read in the tooltip (present tense-number vs. past tense). I defer to @wlach though.

Nice job @MikeLing !

File: ui/partials/perf/graphsctrl.html [hunk 4]
```diff
@@ -78,6 +78,7 @@
         </p>
         <p ng-hide="tooltipContent.revision">Loading revision...</p>
         <p id="tt-t" class="small" ng-bind="tooltipContent.date"></p>
+        <p id="tt-v" ng-show="tooltipContent.retriggers > 0">Retriggers: {{tooltipContent.retriggers}}</p>
```
Comment (wlach) [inline ui/partials/perf/graphsctrl.html]: Could you add a class="small" to this, as we do above? I'd prefer this information be more subtle.

**Comment by wlach** [comment]
Merged manually.

**Pull request closed without merging**